sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.controller.service.ts

```diff
@@ -606,19 +606,17 @@ export class AssessmentControllerService {
         studentScholasticStandingId: assessment.studentScholasticStanding?.id,
       }),
     );
-    // Add unsuccessful scholastic standing to the top of the list, if present.
-    // For unsuccessful scholastic standing, status is always "completed" and
-    // "createdAt" is "submittedDate".
-    if (unsuccessfulScholasticStanding) {
-      history.unshift({
-        submittedDate: unsuccessfulScholasticStanding.createdAt,
+    const unsuccessfulScholasticStandingHistory =
+      unsuccessfulScholasticStandings.map((scholasticStanding) => ({
+        submittedDate: scholasticStanding.submittedDate,
         triggerType: AssessmentTriggerType.ScholasticStandingChange,
+        // For unsuccessful scholastic standing, status is always "completed".
         status: StudentAssessmentStatus.Completed,
-        studentScholasticStandingId: unsuccessfulScholasticStanding.id,
+        studentScholasticStandingId: scholasticStanding.id,
         hasUnsuccessfulWeeks: true,
-      });
-    }
-
+      }));
+    history.push(...unsuccessfulScholasticStandingHistory);
```

Comment:
This part can be conditionally executed if there is one or more scholastic standing.